terraform/main.tf

```diff
@@ -245,4 +245,26 @@ module "webapp-logs-s3" {
   application_name    = "webapp"
 }
 
-
+module "route53" {
+  source                                       = "./modules/route53"
+  root_domain_name                             = var.root_domain_name
+  create_hosted_zone                           = var.create_hosted_zone
```

Comment:
can you do a `count = var.create_hosted_zone` on the module itself?

Reply:
Yes that works